doc/api/globals.md

```diff
@@ -516,6 +516,8 @@ within the browser `var something` will define a new global variable. In
 Node.js this is different. The top-level scope is not the global scope;
 `var something` inside a Node.js module will be local to that module.
 
+> Stability: 3 - Legacy. use [`globalThis`][] instead.
```

Comment:
I believe that this block should be above the type and description as seen here:

https://github.com/nodejs/node/blob/4040a904dedfae8e1a60cc29ed78aab7945d272f/doc/api/globals.md?plain=1#L252-L260

I assume it should be below both the YAML block and `<!-- type=global -->`, but I am not entirely sure. Maybe someone from @nodejs/documentation knows.

Reply:
the shape you suggest offers a healthier doc, I update it this way